.github/workflows/integration-tests.yml

```
@@ -0,0 +1,227 @@
+    
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Run security audit
+        run: npm audit --audit-level=critical || npm audit --audit-level=high --production
+
+      - name: Check for exposed secrets
+        run: |
+          # Check for exposed API keys or secrets (excluding test files and fixtures)
+          # Only fail if we find actual production keys, not test keys
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)